.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ references:
   container_config_node: &container_config_node
     working_directory: ~/project
     docker:
-      - image: circleci/node:14-browsers
+      - image: cimg/node:14-browsers
```

Comment:
I don't know how this is passing but when I try it locally I got
```Error response from daemon: manifest for cimg/node:14-browsers not found: manifest unknown: manifest unknown```

Reply:
the one [suggested](https://github.com/Financial-Times/athloi/pull/83#pullrequestreview-868006979) by ben works